src/server/config/json/user.callback.js

```diff
@@ -17,7 +17,9 @@ export default {
   ERR_USER_NOT_FOUND: 'No such account exists.',
   ERR_UNDEFINED_KEY: 'Key is undefined.',
   ERR_WRONG_PASSWORD: 'The password is not correct.',
+  SUCCESS_ACTIVE_BOOKMARK: 'Success active bookmark.',
```

Comment:
Good I'll update success to succeed!
The title of `activeBookmark` would update as `bookmarkOn`
and `inactiveBookmark` as `bookmarkOff` as same as url.
FYI